features/steps/search_results_page_steps.py

```diff
@@ -24,21 +28,28 @@ def verify_search_results_page_url(context, expected_part_url):
 
 @when("Click on Add to Cart button")
 def click_add_to_cart(context):
-    context.wait.until(EC.element_to_be_clickable(ADD_TO_CART_BTN)).click()
+    sleep(10)
+    context.driver.find_element(*ADD_TO_CART_BTN).click()
 
 
 @when("Confirm Add to Cart button from side navigation")
 def click_add_to_cart(context):
-    context.wait.until(EC.element_to_be_clickable(SIDE_NAV_ADD_TO_CART_BTN)).click()
+    context.driver.find_element(*SIDE_NAV_ADD_TO_CART_BTN).click()
 
 
 @when("Store product name")
 def store_product_name(context):
     context.product_name = context.driver.find_element(*SIDE_NAV_PRODUCT_NAME).text
 
 
+@when("Store product price")
+def store_product_price(context):
+    context.product_price = context.driver.find_element(*PRODUCT_PRICE).text
+
+
 @when('Close side navigation')
 def side_nav_click_add_to_cart(context):
+    sleep(10)
```

Comment:
you can probably wait until SIDE_NAV_PREV_BUTTON is clickable here

Reply:
I tried to use wait here. It didn't work for me.